### PR TITLE
modify topology

### PR DESCRIPTION
**Pull request by Tacha-S**
- using mesh topology instead of star topology.
- remove control node.
- conform pep8 style.
- refuct a part of BLE.

### CODE REVIEW

**Comment by Guvalif** [comment]
Hi Tacha-S,

Before merging these, I want to know how did you test these codes to act perfectly.

**Comment by Tacha-S** [comment]
### ble_node check
1. run `roslaunch plen plen.launch`
2. start up PLENConnect app.
3. see BLE connection logs.
4. restart PLENConnect app.
5. contine to run and catch the message
6. done
### others
1. reboot edison
2. PLEN Eyes light off
3. done

But, This check isn't perfect.
Because, I can't see the message from i2c and accelgylo.

**Comment by musubi05** [comment]
Hi Tacha-S,
Please fix the ros launch file(launch/plen.launch) together; e.g.

#### Before
``` 
<node pkg="plen" name="serialNode" type="serialNode.py"/>
```
#### After
```
<node pkg="plen" name="serial_node" type="serial_node.py"/>
```

**Comment by musubi05** [comment]
Hi Tacha-S,

I found bugs of six_axis_node.py and eyes_node.py .
Please fix that.

File: script/eyes_node.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        self._pwm = mraa.Pwm(pin)
+        self._pwm.period_us(pwm_period_us)
+        self._pwm.enable(True)
+        self._loop = True
+        self._pattern = [0]
+        self._pattern_index = 0
+
+    def set_pattern(self, pattern, loop):
+        # finished pattern or change loop pattern
+        if self._loop ==
```
Comment (musubi05) [inline script/eyes_node.py]: Remove this line.

Comment (musubi05) [inline script/eyes_node.py]: Remove line-19 (if self._loop == ) .

File: script/six_axis_node.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        response.angular.x = accelgyro[3]
+        response.angular.y = accelgyro[4]
+        response.angular.z = accelgyro[5]
+        self.publisher.publish(response)
+
+    def start(self):
+        try:
+            while not rospy.is_shutdown():
+                if self.auto_publish:
+                    self.publish_accelgyros()
```
Comment (musubi05) [inline script/six_axis_node.py]: Fix line-76.

### Before
```
self.publish_accelgyros()
```

### After
```
self.publish()
```

**Comment by musubi05** [comment]
Thank you Techa-S !!

**Pull request closed without merging**